ticdc/ticdc-overview.md

```diff
@@ -41,13 +45,29 @@ Currently, the TiCDC sink component supports replicating data to the following d
 - Databases compatible with MySQL protocol. The sink component provides the final consistency support.
 - Kafka based on the TiCDC Open Protocol. The sink component ensures the row-level order, final consistency or strict transactional consistency.
 
-### Black and white table lists
-
-You can write blacklist and whitelist filtering rules to filter or only replicate all changed data in certain databases or tables. The filtering rules are similar to those of MySQL such as `replication-rules-db` or `replication-rules-table`.
-
 ## Restrictions
 
 To replicate data to TiDB or MySQL, you must ensure that the following requirements are satisfied to guarantee data correctness:
 
 - The table to be replicated has the primary key or a unique index.
 - If the table to be replicated only has unique indexes, each column of at least one unique index is explicitly defined in the table schema as `NOT NULL`.
+
+### Unsupported scenarios
+
+Currently, The following scenarios are not supported:
+
+- The replication of partition tables.
```

Comment:
```suggestion
```
This feature is supported now, we should also update docs-cn

Reply:
updated in https://github.com/pingcap/docs-cn/pull/3552